Makefile

```diff
@@ -42,6 +42,10 @@ prepare-config:
 install-ods-project:
 	cd ods-setup && ./setup-ods-project.sh --namespace $(ODS_NAMESPACE) --reveal-secrets
 
+# AQUA SETUP
+## Setup central "ods" project.
```

Comment:
This text is displayed when you run `make help`. I think it should mention Aqua here.

Reply:
You can find that we just added a better description for this goal in the latest commit